RInAppMessaging/Classes/Services/DisplayPermissionService.swift

```diff
@@ -23,30 +24,36 @@ internal struct DisplayPermissionService: DisplayPermissionServiceType, HttpRequ
 
     func checkPermission(forCampaign campaign: CampaignData) -> DisplayPermissionResponse {
         // In case of error allow campaigns to be displayed anyway
+        let fallbackResponse = DisplayPermissionResponse(display: true, performPing: false)
         let requestParams = [
             Constants.Request.campaignID: campaign.campaignId
         ]
 
-        guard let displayPermissionUrl = configurationRepository.getEndpoints()?.displayPermission,
-            let responseFromDisplayPermission = try? requestFromServerSync(
-                                                        url: displayPermissionUrl,
-                                                        httpMethod: .post,
-                                                        parameters: requestParams,
-                                                        addtionalHeaders: buildRequestHeader()).get().data
-        else {
+        guard let displayPermissionUrl = configurationRepository.getEndpoints()?.displayPermission else {
+            CommonUtility.debugPrint("error: missing endpoint for DisplayPermissionService")
```

Comment:
Out of scope but I don't like CommonUtility. It's awkward. I prefer something like `Logger.debug()` or `Logger.d()`. Maybe we can open source RLogger and use it from IAM.